jetson/src/core/cluster.cpp

```diff
@@ -74,6 +74,32 @@ vector<vector<Point2D>> cluster(vector<Point2D> points, float d) {
     return clustered2d;
 }
 
+vector<Point3D> avgClusterCenters(vector<Point3D> points, float d){
+    vector<Point3D> avgClusterCenters;
+    vector<vector<Point3D>> clusters = cluster(points, d);
+    for(auto cluster : clusters){
+        Point3D sum = (0, 0);
```

Comment:
Copy paste error? Also shouldn't it be `sum(0, 0, 0)`. Idk how to instantiate objects tho